.github/workflows/publish.yml

```diff
@@ -18,18 +18,22 @@ jobs:
       - name: Setup node env 🏗
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: 16
           registry-url: 'https://registry.npmjs.org'
           cache: 'npm'
-          cache-dependency-path: package-lock.json
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Run installation 👨🏻‍💻
         run: npm ci
 
       - name: Run build 🏗
-        run: npm run build
+        run: npm run build-lib
```

Comment:
We should probably make sure that the demo applications can be built when we publish a new version. Or anything speaking against using `npm run build`?